src/types/docs/README.md

```diff
@@ -0,0 +1,325 @@
+# README
+
+## Top-level Schemas
+
+*   [Defender](./defender.md) – `-`
+
+*   [Definitions](./definitions.md) – `-`
+
+*   [Function](./function.md) – `-`
+
+*   [Provider](./provider.md) – `-`
+
+*   [Resources](./resources.md) – `-`
+
+## Other Schemas
+
+### Objects
+
+*   [Autotask](./definitions-definitions-autotask.md) – `undefined#/definitions/autotask`
```

Comment:
Is this `undefined` here ok?

Reply:
These now get removed, however, I won't be referencing this file in the main README, because it contains duplicated references (it lists every reference for every object, even if it's already been referenced before). I do reference the top level properties in the main README, which trickle down to all the other properties anyway.